.github/workflows/ci-sbt.yml

```diff
@@ -43,6 +43,18 @@ jobs:
         with:
           cache: 'yarn'
 
+      - name: Add SSH key for accessing @guardian/private-infrastructure-config
```

Comment:
If we decide this is the right approach we should either write an action for this or adopt one of the public ones that already exist.

Reply:
Would suggest it form part of a [composite action](https://github.com/guardian/cdk/pull/871) and exposed as a `usePrivateInfraConfig` flag resulting in downstream repos having GHA files like:

```yaml
- uses: guardian/cdk@main
  with:
      usePrivateInfraConfig: true
```